third_party/efr32_sdk/efr32_sdk.gni

```diff
@@ -133,13 +139,10 @@ template("efr32_sdk") {
       "${efr32_mcu}=1",
       "${efr32_board}=1",
       "SL_SUPRESS_DEPRECATION_WARNINGS_SDK_3_1",
+      "CHIP_KVS_SECTOR_COUNT=4",
       "CHIP_KVS_BASE_SECTOR_INDEX=((FLASH_SIZE/FLASH_PAGE_SIZE)-(CHIP_KVS_SECTOR_COUNT))",
       "CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI=1",
       "__HEAP_SIZE=0",
-      "SL_CATALOG_FREERTOS_KERNEL_PRESENT=1",
-      "MBEDTLS_THREADING_C=1",
-      "MBEDTLS_THREADING_ALT=1",
-      "SL_THREADING_ALT=1",
```

Comment:
Those change should not be ! Those defines are needed so is CHIP_KVS_SECTOR_COUNT=6 for mg12. Except the new files and includes for wf200. This file should have no change.

Reply:
I re-used the "master" efr32_sdk.gni and added wf200 to it. So the defines you mentioned (and changes based on the MG12/24) is now in sync with master

Reply:
It is checked in now